### ISSUE TITLE
Add wayshot to fedora

### ISSUE THREAD

**Issue by renhiyama** [waycrate/wayshot#83]
Hey there, I am currently setting up Hyprland on Fedora. Everything seems to work, except that I need wayshot in order to take screenshots on hyprland.
Can you devs please add this package to fedora? Here's the link I found: https://fedoraproject.org/wiki/Package_maintainers_wishlist

**Comment by Shinyzenith** [comment]
Hi, we would love to support this but to my knowledge none of the maintainers, including myself,  use fedora. I retired all packaging responsibilities from the repo as it gets hectic and is better handled by experienced individuals.

If there are any interested devs who wanna package this then go for it, I am willing to land patches if needed for this goal. I wonder if wayshot can be packaged as a flatpak, that would be ideal.

**Comment by renhiyama** [comment]
Wayshot could be packaged for flatpak I think. Here's the link https://docs.flatpak.org/en/latest/first-build.html
Its suggested to ask the dev themselves to publish the package to flatpak if possible:
https://docs.flathub.org/docs/for-app-authors/submission/#theres-an-app-that-id-like-to-see-on-flathub-but-im-not-the-developer

**Comment by uncomfyhalomacro** [comment]
Flatpak is better suited for applications. Not for CLI. there is a lot of showstoppers to just use vim or neovim on flatpak (as it is just experimental).

**Comment by Anshi321** [comment]
I’d be interested in working on a Flatpak manifest for wayshot.
Is this still open for contributions? I can share a draft for review.

**Comment by Gigas002** [comment]
I agree with @Shinyzenith and I don't think that package maintainership should be handled in the same repo development happens. I'd prefer to keep distribution-specific packaging out of the source tree.
If any of the users wants to maintain the package for their distro, they should create their own repo and handle it themselves.
